src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -154,32 +153,35 @@ The cluster is automatically given a randomly-generated name. If desired, change
 
 ## Step 8. Select the CockroachDB version
```

Comment:
@jaiayu PTAL at this section, which has expanded from when you previously took a look. UI verified using the Education production org, which is not enrolled in the new UI.